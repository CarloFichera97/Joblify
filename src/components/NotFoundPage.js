import React from "react";
import { Link, Redirect } from "react-router-dom";
import Header from "./Header";
window.onload = function () {
  document.querySelector(".cont_principal").className =
    "cont_principal cont_error_active";
};

const NotFoundPage = () => (
  <div>
    {" "}
    <Header />
    <div className="box-layout_notfound">
      <div className="face">
        <div className="band">
          <div className="green"></div>
          <div className="white"></div>
          <div className="red"></div>
        </div>
        <div className="eyes"></div>
        <div className="dimples"></div>
        <div className="mouth"></div>
      </div>
      <Link to="/dashboard" className="not_found_title">
        ERROR 404 - The Developer probably fell asleep
      </Link>
    </div>
  </div>
);

export default NotFoundPage;
