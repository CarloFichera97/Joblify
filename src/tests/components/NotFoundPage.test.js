import React from "react";
import NotFoundPage from "../../components/NotFoundPage";
import { shallow } from "enzyme";

test("Should render Not Found Page correctly", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
