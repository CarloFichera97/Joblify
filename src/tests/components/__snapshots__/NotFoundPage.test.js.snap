// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Not Found Page correctly 1`] = `
<div>
   
  <Header />
  <div
    className="box-layout_notfound"
  >
    <div
      className="face"
    >
      <div
        className="band"
      >
        <div
          className="green"
        />
        <div
          className="white"
        />
        <div
          className="red"
        />
      </div>
      <div
        className="eyes"
      />
      <div
        className="dimples"
      />
      <div
        className="mouth"
      />
    </div>
    <Link
      className="not_found_title"
      replace={false}
      to="/dashboard"
    >
      ERROR 404 - The Developer probably fell asleep
    </Link>
  </div>
</div>
`;
